client/app/vps/backup-storage/vps-backup-storage.controller.js

```diff
@@ -0,0 +1,90 @@
+class VpsBackupStorageCtrl {
+    constructor ($stateParams ,$translate, CloudMessage, VpsActionService ,VpsService) {
+        this.serviceName = $stateParams.serviceName;
+        this.$translate = $translate;
+        this.CloudMessage = CloudMessage;
+        this.VpsActionService = VpsActionService;
+        this.VpsService = VpsService;
+
+        this.loaders = {
+            init: false,
+            information: false
+        };
+        this.backup = {};
+
+    }
+
+    $onInit () {
+        this.loadInformation ();
+        this.loadBackup();
+    }
+
+    loadInformation () {
+        this.loaders.information = true;
+        this.VpsService.getBackupStorageInformation(this.serviceName)
+            .then((backupInfo) => {
+                this.backup.information = backupInfo;
+                if (backupInfo.activated === true && backupInfo.quota) {
+                    if (backupInfo.usage === 0) {
+                        backupInfo.usage = {
+                            unit: "%",
+                            value: 0
+                        };
+                    }
+                    this.backup.use = backupInfo.usage.value * backupInfo.quota.value / 100;
+                }
+            })
+            .catch((err) => { this.CloudMessage.error(err) })
+            .finally(() => {
+                this.loaders.information = false;
+            });
+    }
+
+    loadBackup () {
+        this.loaders.init = true;
+        this.VpsService.getBackupStorageTab(this.serviceName)
+            .then(data => { this.backup.table = data })
+            .catch(err => this.CloudMessage.error(err))
+            .finally(() => { this.loaders.init = false });
+    }
+
+    getStatus (isApplied) {
```

Comment:
This is an inversion of the controller/view dynamic, 

It's an addon you made, code before was fine, see -
<td class="center">
  <span data-ng-show="backupStorage.isApplied" class="badge badge-success" data-translate="vps_tab_backup_storage_table_ip_enable"></span>
  <span data-ng-hide="backupStorage.isApplied" class="badge badge-important" data-translate="vps_tab_backup_storage_table_ip_disable"></span>
</td>

Reply:
I cannot apply this to listView

Reply:
Yes I can :-), by using a template
```
return `
    <td class="center">
        <span data-ng-show="$row.isApplied" class="badge badge-success" data-translate="vps_tab_backup_storage_table_ip_enable"></span>
        <span data-ng-hide="$row.isApplied" class="badge badge-important" data-translate="vps_tab_backup_storage_table_ip_disable"></span>
</td>`;
```
Here's a long explication of some other wrong stuff 
-Naming a function get with a parameter is NOT a real get.
-In fact what you are doing is `VpsBackupStorageCtrl.getStatus(...)` this should return status of the controller, because it's the parent of your function
-When you return a formatted String you're not using the most obvious feature of angular-bind
-In your Model your Status is status is the boolean isApplied, what you are retrieving is a formatted string for the view

Reply:
done